oecd.csl

```diff
@@ -0,0 +1,293 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>OECD</title>
+    <title-short>OECD </title-short>
+    <id>http://www.zotero.org/styles/oecd</id>
+    <link href="http://www.zotero.org/styles/oecd" rel="self"/>
+    <link href="http://www.zotero.org/styles/deutsche-gesellschaft-fur-psychologie" rel="template"/>
+    <link href="https://www.oecd.org/site/eduimhe08/guidelinesforauthors.htm" rel="documentation"/>
```

Comment:
@POBrien333, do you know if the style matches what is described in https://www.iea.org/publications/freepublications/publication/IEAOECD_style_guide.pdf. That seems to be a much better documentation link (that style guide seems to be both for the IEA and the OECD, by the way).

The current link is for a 2008 Institutional Management in Higher Education (IMHE) conference, and the IMHE program seems to have been canceled in 2016 (see http://www.oecd.org/education/imhe/: "In November 2015, the IMHE Governing Board agreed not to renew the programme's mandate which expired on 31 December 2016. As a result, the IMHE programme closed at the end of 2016.").

Reply:
@POBrien333 see Rintze's questions above, please

Reply:
Hi. I missed this one.
The two guidelines seem to be matching, but there is one small difference: in-text et al. is in italics for the "OECD" guidelines.

Since IMHE is dead now anyway and the IAE guidelines are way better documented, let's make the style for that then.